Source/Scene/Model/Model.js

```diff
@@ -1167,6 +1176,17 @@ Object.defineProperties(Model.prototype, {
         this.resetDrawCommands();
       }
       this._debugWireframe = value;
+
+      //Warning for improper setup of debug wireframe
+      if (
+        this._debugWireframe === true &&
+        this._enableDebugWireframe === false
+      ) {
```

Comment:
Since tilesets use `Model` for rendering the tile contents, this produces 2 warnings. See [this Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=bVNRb9owEP4rFk9BojYVrSYxitbBwyqta9WiVZPy4thHOM2xI/sCpdP+++wkVNDxRO7u+767+84Iwb42aDTakmlJkinXeIKwZ27NfvxasIca7DIVaudJminbENVTIXa73SW3e8VLtxUBCYR2SCTQIqEk3EIQE31R9OK8liXkVjkbiG0RduDZDbOwYwsI2FT8Z5vL8oFq44WzJNGCzwcj9ie3jBF4HzOP3m1Rg58eiMqDJHhx3uhVB8mGo9z+HX7ObW67VjwosMBL4wrgGmrarCDQbRnBgXpWHId8A8cseCWwOusbdckumCxXaCDc2VCDIufv8RVt6tgtiId82ureaTBRvRdVZwVa9RbZDtDJUIIAnRp1MgBQ1prTePNuyJ2zTxDiGRXwtXfVbYiwO519up5cTZI1jIGVhYElFE35gh7WXlYwZWtpArR1/aGSjOktPTG09lhhe2sutc76eTvn+4DH8+h9vFqFAThtwGbrxipCZ1k27E7ba745V61cljLssPuoi472/xb14oN6RFKbJ2lL6AmMjfl4dPi+GPPr9+AwSuEam7jP9QY8cB+FmsAEu+LjDjpMP2n8dtXBaDALtDcwPwh9wSr9DZLdGeeCoKpNfHxBFI36HRuoEBIxQWfimDrTuGWob848b6aMDCFW1o0xz/gG+WA+ExH/H9W4dvOHLXgj9wm2uZx/75Kc85mI4XkmOWcK6T8o/wM)

To filter out this case, I think we should add a check to this if statement (along with the one in the constructor)

```
this.type === ModelType.GLTF
```

This way this warning only shows up for individual models